src/runtime_src/core/pcie/emulation/cpu_em/generic_pcie_hal2/shim.cxx

```diff
@@ -493,22 +491,29 @@ namespace xclcpuemhal2 {
         }
 
         //fclose (stdout);
-        if(r == -1){std::cerr << "FATAL ERROR : child process did not launch" << std::endl; exit(1);}
+        if (r == -1)
+        {
+          std::cerr << "FATAL ERROR : child process did not launch" << std::endl;
+          exit(1);
+        }
         exit(0);
       }
     }
     sock = new unix_socket("EMULATION_SOCKETID");
   }
 
-  void CpuemShim::getCuRangeIdx() {
+  void CpuemShim::getCuRangeIdx()
+  {
     std::string instance_name = "";
-    for (const auto& kernel : m_xclbin.get_kernels()) {
+    for (const auto &kernel : m_xclbin.get_kernels())
+    {
       // get properties of each kernel object
-      const auto& props = xrt_core::xclbin_int::get_properties(kernel);
+      const auto &props = xrt_core::xclbin_int::get_properties(kernel);
```

Comment:
bad style, should `const auto&`  Same goes for all arguments to functions.  I don't  care enough since this is not code I edit.